src/dtgtk/thumbnail.c

```diff
@@ -197,10 +197,13 @@ static void _image_get_infos(dt_thumbnail_t *thumb)
 
 static gboolean _thumb_expose_again(gpointer user_data)
 {
-  if(!user_data || !GTK_IS_WIDGET(user_data)) return FALSE;
+  dt_thumbnail_t *thumb = (dt_thumbnail_t *)user_data;
+  gpointer w_image = thumb->w_image;
+  if(!thumb) return FALSE;
```

Comment:
this test is too late as just above you use the pointer to access w_image.

I would keep:
`if(!user_data)) return FALSE;`
as the first line on this routine.

Reply:
You are right. A fix is incoming.